_articles/best-practices.md

```diff
@@ -47,7 +47,7 @@ For example, @lord discovered that having a project vision helped him figure out
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars.githubusercontent.com/lord?s=180" class="pquote-avatar" alt="avatar">
-  I fumbled it. I didn't put in the effort to come up with a complete solution. Instead of an half-assed solution, I wish I had said "I don't have time for this right now, but I'll add it to the long term nice-to-have list."
```

Comment:
a

Reply:
Yeah instead of a half-assed sounds better. Thanks, mate